ethereum/forkchoice/src/main/java/tech/pegasys/teku/ethereum/forkchoice/ProtoArrayIndices.java

```diff
@@ -31,10 +31,12 @@ public void add(final Bytes32 blockRoot, final int nodeIndex) {
     rootIndices.put(blockRoot, nodeIndex);
   }
 
+  @SuppressWarnings("deprecation")
   public Optional<Integer> get(final Bytes32 root) {
     return Optional.ofNullable(rootIndices.getOrDefault(root, null));
   }
 
+  @SuppressWarnings("deprecation")
   public void remove(final Bytes32 root) {
     rootIndices.remove(root);
```

Comment:
This one can just switch to using:
```suggestion
  public void remove(final Bytes32 root) {
    rootIndices.removeInt(root);
```